api/src/main/java/org/openmrs/logging/OpenmrsConfigurationFactory.java

```diff
@@ -103,6 +114,33 @@ protected String[] getSupportedTypes() {
 		return SUFFIXES;
 	}
 	
+	public List<File> getConfigurationFiles() {
+		List<File> configurationFiles = new ArrayList<>();
+
+		// Get all extensions except for the wildcard extension
+		List<String> extensionList = new ArrayList<>();
+		for (String suffix : SUFFIXES) {
+			if (!suffix.equals("*")) {
+				extensionList.add(suffix.substring(1)); // Add the suffix without the leading "."
+			}
+		}
+		String[] extensions = extensionList.toArray(new String[] {});
```

Comment:
Couldn't this be done statically, e.g.,

```
public static final String[] EXTENSIONS = Arrays.stream(SUFFIXES).filter(s -> !s.equals("*")).map(s -> s.substring(1)).toArray(String[]::new);
```

The suffixes shouldn't really change at runtime....

Reply:
Made this change, let me know how this looks @ibacher